src/libs/TransactionUtils.js

```diff
@@ -15,7 +15,7 @@ Onyx.connect({
         if (!val) {
             return;
         }
-        allTransactions = val;
+        allTransactions = _.pick(val, (transaction) => transaction);
```

Comment:
isn't filter makes more sense as pick used for picking the specific properties from the object?

Reply:
allTransactions is an object so we are picking the not null here. when using a filter it will return as an array